client/post-editor/index.js

```diff
@@ -18,6 +18,7 @@ export default function () {
 		'/post/:site?/:post?',
 		siteSelection,
 		controller.gutenberg,
+		controller.classic,
```

Comment:
Another option would be to use the `maybeRedirect` function.

Reply:
I've tried updating the `maybeRedirect` function in its current place though that results in a flash of the editor being loaded before the actual redirect to WP admin takes affect. 

I could go about pulling it out of the `startEditing` function and moving it earlier in the middleware chain. Is it worth doing that now? Is it something that could be done if/when we move the redirect logic from all the other links into one location?

Reply:
I think the current solution is good 👍